src/Framework/PaymentGateways/Contracts/PaymentGateway.php

```diff
@@ -2,10 +2,23 @@
 
 namespace Give\Framework\PaymentGateways\Contracts;
```

Comment:
How is this a contract? It's not an interface.

Reply:
resolved 6e8a5a13